### DISCUSSION TITLE
Requirement Installation

### DISCUSSION THREAD

**Discussion by amsitlab** [vlang/v#15942] [Installation Issues]
Whats minimal requirement to Install V?
Gcc/Clang version?
Architecture?
etc?

**Comment by spytheman** [comment]
The only basic requirement, is a C compiler, that supports the C99 standard.

For the supported platforms (Linux, Windows, macos), V can download a prebuilt copy of tcc that is tested to work.

I have not tested the absolute minimum, but currently I have gcc-8 and clang-7 (on Ubuntu 20.04), and I can compile V itself with them: (`v -cc gcc-8 -o vgcc cmd/v` and `v -cc clang-7 -o vgcc cmd/v`).

For some of the examples (mysql, graphical ones), you will also need other development libraries, see <https://github.com/vlang/v/blob/master/README.md> .
